src/SamplesApp/UITests.Shared/Windows_UI_Xaml_Controls/TextBlockControl/TextBlock_Special_Character.xaml.cs

```diff
@@ -3,8 +3,8 @@
 
 namespace Uno.UI.Samples.Content.UITests.TextBlockControl
 {
-	[SampleControlInfo("TextBlockControl", "TextBlock_Special_Character", description: "Sample TextBlock with Special Character")]
-	public sealed partial class TextBlock_Special_Character : UserControl
+	[SampleControlInfo(description: "Sample TextBlock with Special Character")]
```

Comment:
If you're about to change those, you should also update the type of the attribute to `Sample`